### PR TITLE
Remove xcmp-handler fees rpc

### PR DESCRIPTION
**Pull request by nvengal**
Deprecated by `AutomationTime#queryFeeDetails`. Removing the endpoint from the node to avoid any confusion by partners on which fee rpc is the source of truth.

### CODE REVIEW

File: node/src/chain_spec/mod.rs [hunk 1]
```diff
@@ -19,6 +19,7 @@ pub mod oak;
 #[cfg(feature = "turing-node")]
 pub mod turing;
 
+#[allow(dead_code)]
```
Comment (nvengal) [inline node/src/chain_spec/mod.rs:22]: Unrelated changes to clean up compiler warnings. This is not actually dead code, it is only present under some features. Rather than be pedantic about the features let's just suppress the warning.

**Review by chrisli30 — APPROVED** [review]
Yes, this is a good call. The changes look good to me.

@imstar15 FYI

**Review by imstar15 — APPROVED** [review]
looks good to me!
